src/osd/ReplicatedPG.cc

```diff
@@ -4351,8 +4352,8 @@ int ReplicatedPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
           if (result >= 0) {
 	    if (maybe_create_new_object(ctx)) {
               ctx->mod_desc.create();
+	      t->touch(soid);
```

Comment:
@athanatos I will remove both commits.  Maybe we won't see the bugs since we didn't backport all of the op tracking changes.